aspnetcore/release-notes/aspnetcore-9/samples/PAR/Program.cs

```diff
@@ -4,24 +4,27 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // <snippet_1>
-builder.Services
-    .AddAuthentication(options =>
-    {
-        options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-        options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
-    })
-    .AddCookie()
-    .AddOpenIdConnect("oidc", oidcOptions =>
-    {
-        // Other provider-specific configuration goes here.
-
-        // The default value is PushedAuthorizationBehavior.UseIfAvailable.
-        oidcOptions.PushedAuthorizationBehavior = PushedAuthorizationBehavior.Disable;
-    });
+ builder.Services
+     .AddAuthentication(options =>
+     {
+         options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+         options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
+     })
+     .AddCookie()
+     .AddOpenIdConnect("oidc", oidcOptions =>
+     {
+         // Other provider-specific configuration goes here.
+ 
+         // The default value is PushedAuthorizationBehavior.UseIfAvailable.
+ 
+         // 'OpenIdConnectOptions' does not contain a definition for 'PushedAuthorizationBehavior'
+         // and no accessible extension method 'PushedAuthorizationBehavior' accepting a first argument
+         // of type 'OpenIdConnectOptions' could be found
+         oidcOptions.PushedAuthorizationBehavior = PushedAuthorizationBehavior.Disable;
+     });
```

Comment:
@halter73  @jennyf19 @brentschmaltz I've installed 8.0.400-preview.0.24324.5 and it can't find `PushedAuthorizationBehavior`. I then installed a 9.0.100-rc.1.24409.1 and it's not there either. What version do I need?

Reply:
Did you forget to also use a preview release of the Microsoft.AspNetCoreAuthentication.OpenIdConnect NuGet package? That was the mistake I made at first when I [tried the preview builds](https://jmdc.dev/posts/getting-started-with-par/).

Reply:
> Did you not also use a preview release of the Microsoft.AspNetCoreAuthentication.OpenIdConnect NuGet package? That was the mistake I made at first when I [tried the preview builds](https://jmdc.dev/posts/getting-started-with-par/).

In VS, I've got `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json` as my NuGet source.

Should I replace the code I have now with your [tried the preview builds](https://jmdc.dev/posts/getting-started-with-par/). code that doesn't explicitly call `PushedAuthorizationBehavior` ?

```
builder.Services.AddAuthentication(opt =>
    {
        opt.DefaultScheme = "cookie";
        opt.DefaultChallengeScheme = "oidc";
    })
    .AddOpenIdConnect("oidc", opt =>
    {   
        opt.Authority = "https://demo.duendesoftware.com";
        opt.ClientId = "interactive.confidential";
        opt.ClientSecret = "secret";
        opt.ResponseType = "code";
        opt.DisableTelemetry = true;
    })
    .AddCookie("cookie");
builder.Services.AddAuthorization();
``

Reply:
What version of the NuGet package did you install?

Reply:
> What version of the NuGet package did you install?

Arg, I had NuGet sources in the wrong order. It's building now. Thanks for your help.

Reply:
No problem - glad it turned out to be an easy fix!